ios/ThreadManager.m

```diff
@@ -53,8 +53,8 @@ -(void) checkAndSendEvent:(NSString*)name body:(id)body
     : name;
 
   // Bundled JavaScript is placed within the resources directory.
-  NSURL *bundledThreadURL = [[NSBundle mainBundle] URLForResource:[NSString stringWithFormat:@"%@.thread", [jsFileSlug lastPathComponent]] withExtension:@"jsbundle"];
-
+  //NSURL *bundledThreadURL = [[NSBundle mainBundle] URLForResource:[NSString stringWithFormat:@"%@.thread", [jsFileSlug lastPathComponent]] withExtension:@"jsbundle"];
```

Comment:
We should delete this line instead of commenting it out.

Reply:
Thanks!

https://github.com/ExodusMovement/react-native-threads/pull/10/commits/400bffefdcac772e99a279b44545c9f10328ad7d